handler/pair.go

```diff
@@ -0,0 +1,45 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"path"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cloudradar-monitoring/plexus/api"
+)
+
+// Pair godoc
+// @Summary Gets the pairing code info.
+// @Tags session
+// @Produce application/json
+// @Param code path string true "code"
+// @Security BasicAuth
+// @Success 200 {object} api.URLResponse
+// @Failure 401 {object} api.Error
+// @Failure 404 {object} api.Error
+// @Router /pairing/{code} [get]
+func (h *Handler) Pair(rw http.ResponseWriter, r *http.Request) {
+	h.lock.Lock()
+	defer h.lock.Unlock()
+	id, ok := h.getSessionID(mux.Vars(r)["code"])
+	if !ok {
+		api.WriteJSONError(rw, http.StatusNotFound, "Code not found")
+		return
+	}
+
+	session, ok := h.checkSessionAuthentication(rw, r, id)
+	if !ok {
+		return
+	}
```

Comment:
I think the pairing shouldn't require session authentication. As this makes creating a connection with the user more difficult. See issue description:
> , I want to be able to guide the person who needs my help through the process of starting the agent. This must be possible on a phone call.

I'd say only the pairing code should be requried.